container/test-impl-base/src/main/java/org/jboss/arquillian/container/test/impl/RunModeUtils.java

```diff
@@ -38,7 +40,7 @@
    private RunModeUtils() { }
    
    /**
-    * Check is this should run as client.
+    * Returns if the given test should run as client.
```

Comment:
Well, for me the verification process means that the method usually doesn't return anything and if the conditions isn't accomplished (the test shouldn't be ran as a client test), then it is wrong and for example an exception is thrown. But this is only my point of view :-). 
I can rephrase it if it makes more sense for you ;-).